eng/Signing.props

```diff
@@ -1,6 +1,17 @@
 <Project>
-  <ItemGroup>
+  <ItemGroup Condition="'$(SignFilesToBundle)' != 'true'">
     <ItemsToSign Include="$(ArtifactsPackagesDir)**\*.zip"/>
+    <ItemsToSign Include="$(ArtifactsDir)bundledtools\**\*.exe" />
```

Comment:
yeah, same thing with our usual tools. In this case it's even nastier because if we want them to actually work, we need to sign + harden + notarize. These are only available in yaml now...

Reply:
Meaning we would need to add a step to the yaml pipeline definition before we do the artifact upload?

Reply:
Meaning... I'm not quite sure. I'm not familiar with OSX signing that much. If only executables need to get signed + entitled then the tool bundles should be somewhat easy. If every DLL and dylib needs to get signed, then we have some issues as arcade signing doesn't support OSX signing other than PKG afaik. I'd need to revisit that.

Reply:
For what it's worth, I am able to run the OSX binaries produced by this patch on my mac with SIP enabled.